plugins/cuda/Makefile

```diff
@@ -10,7 +10,7 @@ PLUGIN_INCLUDE  	+= -iquote../../
 
 COMPEL := ../../compel/compel-host
 
-CC := gcc
+CC := $(CROSS_COMPILE)$(HOSTCC)
```

Comment:
We can remove this line. `CC` is defined in `tools.mk`:
https://github.com/checkpoint-restore/criu/blob/a11e944996d4a71ed275cc12bd6eefa795015100/scripts/nmk/scripts/tools.mk#L12